lib/rules/attribute-order.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @fileoverview enforce ordering of attributes
+ * @author Erin Depew
+ */
+'use strict'
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+function getPosition (attribute, attributeOrder) {
+  const name = attribute.key.name
+  if (name === 'is') {
+    return attributeOrder.indexOf('DEFINITION')
+  } else if (name === 'for') {
+    return attributeOrder.indexOf('LIST_RENDERING')
+  } else if (name === 'if' || name === 'else-if' || name === 'else' || name === 'show' || name === 'cloak') {
+    return attributeOrder.indexOf('CONDITIONALS')
+  } else if (name === 'pre' || name === 'once') {
+    return attributeOrder.indexOf('RENDER_MODIFIERS')
+  } else if (name === 'id') {
+    return attributeOrder.indexOf('GLOBAL')
+  } else if (name === 'ref' || name === 'key' || name === 'slot') {
+    return attributeOrder.indexOf('UNIQUE')
+  } else if (name === 'model') {
+    return attributeOrder.indexOf('BINDING')
```

Comment:
`v-bind:something` and `:something` should also be marked as binding I think, not only `v-model`.
Also what if someone adds property `model="asd"`? I think it would mark it as binding and not OTHER_ATTR. You should probably check if `attribute.directive` is true in all cases where you want to detect `v-` attribute.